packages/firebase_messaging/README.md

```diff
@@ -83,7 +83,9 @@ By default background messaging is not enabled. To handle messages in the backgr
    import io.flutter.plugin.common.PluginRegistry.PluginRegistrantCallback;
    import io.flutter.plugins.GeneratedPluginRegistrant;
```

Comment:
There's no need to import `GeneratedPluginRegistrant`

Reply:
yes @rhisiart-cufflink 
we don't need it.
Thanks